src/kitsController.ts

```diff
@@ -317,7 +317,7 @@ export class KitsController {
      *
      * Always returns immediately.
      */
-    private static _startPruneOutdatedKitsAsync(cmakeProject: CMakeProject) {
+    private static _startPruneOutdatedKitsAsync(cmakePath: string) {
```

Comment:
Why are so many functions being changed to pass in cmakePath instead of cmakeProject when you can get the path from the project?

Is it for tests?